ClientApp/viking-vault/src/display_user_list/AddMoney.css

```diff
@@ -0,0 +1,63 @@
+.modal.open {
+    display: block;
+    opacity: 1;
```

Comment:
Why is this opacity used? I see that when the modal is closed you simply set it's `display` to `none`